oci/output.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+GREEN = "\033[1;32m"
+RED = "\033[1;31m"
+WHITE = "\033[1;37m"
+RESET = "\033[0m"
+
+def info(s, *args):
+    _output(s, *args)
+
+def failure(s, *args):
+    _output(s, *args, color=RED)
+
+def success(s, *args):
+    _output(s, *args, color=GREEN)
```

Comment:
This should be:

    _output(s, args, color=GREEN)

```
>>> from oci import output
>>> output.info("msg")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "oci/output.py", line 9, in info
    _output(s, *args)
TypeError: _output() takes at least 2 arguments (1 given)
```
This tell us that we need a simple test like:

    def test_info(capsys):
        output.info("msg")
        captured = capsys.readouterr()
        assert captured.out == "msg\n"

And same for other colors.

Reply:
Done